lib/src/ui/watch/watch_screen.dart

```diff
@@ -0,0 +1,200 @@
+          slivers: [
+            CupertinoSliverNavigationBar(
+              trailing: CupertinoButton(
+                padding: EdgeInsets.zero,
+                child: isSoundMuted
+                    ? const Icon(CupertinoIcons.volume_off)
+                    : const Icon(CupertinoIcons.volume_up),
+                onPressed: () =>
+                    ref.read(muteSoundPrefProvider.notifier).toggle(),
+              ),
+            ),
+            CupertinoSliverRefreshControl(
+              onRefresh: () => ref.refresh(streamerListProvider.future),
+            ),
+            SliverSafeArea(
+              top: false,
+              sliver: SliverPadding(
+                padding: Styles.verticalBodyPadding,
+                sliver: SliverList(
+                  delegate: SliverChildListDelegate(
+                    [const _WatchTvWidget(), StreamerWidget()],
+                  ),
+                ),
+              ),
+            )
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class _WatchTvWidget extends ConsumerWidget {
+  const _WatchTvWidget();
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final pieceSet = ref.watch(pieceSetPrefProvider);
+    final currentTab = ref.watch(currentBottomTabProvider);
+    final tvStream = currentTab == BottomTab.watch
+        ? ref.watch(tvStreamProvider)
+        : const AsyncLoading<FeaturedPosition>();
+    final featuredGame = ref.watch(featuredGameProvider);
+    return ListSection(
+      hasLeading: true,
+      header: const Text('Watch Top Games'),
+      onHeaderTap: () => Navigator.of(context).push<void>(
+        MaterialPageRoute(builder: (context) => const TvScreen()),
+      ),
+      children: [
+        tvStream.when(
+          data: (position) {
+            final boardData = BoardData(
+              interactableSide: InteractableSide.none,
+              orientation: featuredGame?.orientation.cg ?? Side.white,
+              fen: position.fen,
+              lastMove: position.lastMove?.cg,
+            );
+            return Center(
+              child: ConstrainedBox(
+                constraints: BoxConstraints.tight(const Size.square(350)),
+                child: GameBoardLayout(
+                  topPlayer: kEmptyWidget,
+                  bottomPlayer: kEmptyWidget,
+                  boardData: boardData,
+                  boardSettings: BoardSettings(
+                    animationDuration: Duration.zero,
+                    pieceAssets: pieceSet.assets,
+                  ),
+                ),
+              ),
+            );
+          },
+          error: (err, stackTrace) {
+            debugPrint(
+              'SEVERE: [WatchTvWidget] could not load tv stream; $err\n$stackTrace',
+            );
+            return const EmptyBoard(errorMessage: 'Could not load TV Stream');
+          },
+          loading: () => const EmptyBoard(),
+        ),
+      ],
+    );
+  }
+}
+
+class EmptyBoard extends StatelessWidget {
```

Comment:
If it is only used once, I'd rather not have a widget but the code directly in place. Here the size and board data have some duplication.